src/pages/index.tsx

```diff
@@ -2,45 +2,33 @@
 
 import { type NextPage } from "next";
 import { api } from "~/utils/api";
+import Topbar from "~/pages/topbar/Topbar";
+import ParticipantsForm from "~/pages/components/ParticipantsForm"
+import { useState } from 'react';
+
 
 const Home: NextPage = () => {
 
   // API Examples
   const postGroup = api.group.postGroup.useMutation();
-  const postSecretSanta = () => {
-    postGroup.mutate({ list: [ 'Sara', 'Gina' ] });
+  // const postSecretSanta = () => {
+  //   postGroup.mutate({ list: [ 'Sara', 'Gina' ] });
+  // };
+  const postSecretSanta = (userData) => {
+    console.log("Invoke func postSecretSanta .. data: " + userData)
+    postGroup.mutate({ list: userData });
   };
 
+
   const getGroup = api.group.getGroup.useQuery({ groupId: "42" });
   const getDraw = api.draw.getDraw.useQuery({ passphrase: "sara-loves-christmas" });
 
   return (
-    <main>
-      <h1 className="text-2xl">Secret Santa</h1>
-      <div>
-        <button className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded-full" onClick={postSecretSanta} disabled={postGroup.isLoading}>
-          POST group (Mutate) - Input(list: string[])
-        </button>
-
-        <p className="text-1xl text-red-500">
-          {postGroup.error && <p>Something went wrong! {postGroup.error.message}</p>}
-          {postGroup.data ? "groupId: " + postGroup.data.groupId : "Loading tRPC query..."}
-        </p>
-      </div>
-      <h1 className="text-2xl">Mock Data</h1>
-      <div>
-        <p className="text-1xl text-white">
-          GET group (Query) - Input(groupId):
-          {getGroup.data ? getGroup.data.group.map(el => <div>{" - giver: " + el.giver}, {"passphrase: " + el.passphrase}</div>)  : "Loading tRPC query..."}
-        </p>
-        <p className="text-1xl text-white">
-          GET draw (Query) - Input(passphrase)
-          {getDraw.data ? " - reciver: " + getDraw.data.reciver : "Loading tRPC query..."}
-        </p>
-      </div>
-    </main>
-  );
+    <div className="page">
+      <Topbar />
+      <ParticipantsForm post={postSecretSanta}/>
+    </div>
+  )
 };
 
-export default Home;
-
+export default Home;
```

Comment:
Let's agree on this. Do we do newlines end of file or not?

Reply:
ok. + newlines